plugins/modules/system/lvg.py

```diff
@@ -249,6 +256,31 @@ def main():
         devs_to_remove = list(set(current_devs) - set(dev_list))
         devs_to_add = list(set(dev_list) - set(current_devs))
 
+        if current_devs:
+            if state == 'present' and pvresize:
+                for device in current_devs:
+                    pvdisplay_cmd = module.get_bin_path('pvdisplay', True)
+                    pvdisplay_ops = ["--units", "b", "--columns", "--noheadings", "--nosuffix"]
+                    pvdiplay_cmd_device_options = [pvdisplay_cmd, device] + pvdisplay_ops
+                    rc, dev_size, err = module.run_command(pvdiplay_cmd_device_options + ["-o", "dev_size"])
+                    dev_size = int(dev_size.replace(" ", ""))
+                    rc, pv_size, err = module.run_command(pvdiplay_cmd_device_options + ["-o", "pv_size"])
+                    pv_size = int(pv_size.replace(" ", ""))
+                    rc, pe_start, err = module.run_command(pvdiplay_cmd_device_options + ["-o", "pe_start"])
+                    pe_start = int(pe_start.replace(" ", ""))
+                    rc, vg_extent_size, err = module.run_command(pvdiplay_cmd_device_options + ["-o", "vg_extent_size"])
+                    vg_extent_size = int(vg_extent_size.replace(" ", ""))
+                    if (dev_size - (pe_start + pv_size)) > vg_extent_size:
+                        if module.check_mode:
+                            changed = True
+                        else:
+                            pvresize_cmd = module.get_bin_path('pvresize', True)
```

Comment:
probably cosmetic, but i would move this pvresize_cmd next to pvdisplay_cmd

Reply:
adopted your suggestion. Also caught a typo on "pvdiplay_cmd_device_options ". There was a missing "s"

Reply:
thanks